.travis.yml

```diff
@@ -1,8 +1,9 @@
 sudo: false
 language: python
 before_install:
+- nvm install node && nvm use node
 - node --version
-- npm install --python=python2 -g dredd
+- npm install --python=python2 -g dredd --no-optional
```

Comment:
I assume the `--python=python2` was for Protagonist/Drafter's configure script. Since you've added `--no-optional` and pure drafter JS would be used, you may be able to remove `--python=python2`.